CharacterCreation/03_Feats.txt

```diff
@@ -759,17 +759,16 @@ Melee Expertise effectively has no prerequisite.
 
 0:	+10m to range R2. If this would cause R2 to be greater than R3, 
 	also increase R3 to match the new R2.
-
 12:	May use Weapons - Carbine (Dex) in place of Acrobatics on checks to maintain 
-	balance and negate combat penalties associated with a vehicle or mount 
-	you are on board.
-24:	+25 Weapon Reflex while onboard a vehicle or mount and for 3 rounds 
+	balance and negate combat penalties associated with a moving vehicle or mount 
+	that you are on board.

```

Comment:
"a moving vehicle or mount that you are onboard" is really awkward, passive phrasing :(

Reply:
Yes you are correct. This is what happens when bits are patched thoughtlessly instead of redone from the ground up with the new assumptions. X|